crates/anvil/src/engine/TODO.md

```diff
@@ -0,0 +1,5 @@
+Decisions to make
+* pass multiple ports into spawn method?
+* have serve method do both EthAPI and EngineAPi or separate methods?
+* can skip JWT auth?
+* have HttpEthRpcHandler and PubSubEthRpcHandler take in both ethapi and engineapi or have duplicate methods
```

Comment:
the HttpEngineHandler yes, this then just reuses the `HttpEthRpcHandler` internally for a subset of the support endpoints.

there are no pubsub endpoints on the engine server

https://github.com/ethereum/execution-apis/blob/main/src/engine/common.md#underlying-protocol

this means we either need a new handler for it, that simply rejects subscription requests, or find away to live without the `PubSubRpcHandler` trait for WS.